qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -56,5 +62,56 @@ def __init__(
         self.hub_priority = hub_priority
         self.group_priority = group_priority
         self.project_priority = project_priority
+        self.job_id = job_id
 
         super().__init__(**kwargs)
+
+    def __repr__(self) -> str:
+        """Return the official string representation of QueueInfo.
+
+        Returns:
+            a string representation of QueueInfo.
+        """
+        queue_info = []
+        for attr, value in self.__dict__.items():
+            if isinstance(value, datetime):
+                value = value.isoformat()
+            elif value in [status.value for status in ApiJobStatus]:
+                value = api_status_to_job_status(ApiJobStatus(value)).value
+            queue_info.append("{}='{}'".format(attr, self._get_value(value)))
```

Comment:
Since the point of the custom `__repr__` is to be user friendly, can we make it more so by grouping related fields together in order? So instead of 
```
<QueueInfo(_status='job is queued', estimated_complete_time='2020-01-17T22:59:25.907000+00:00', project_priority='1.0', job_id='5e22013e69f2d4001830bbbd', estimated_start_time='2020-01-17T22:58:54.507000+00:00', hub_priority='1.0', group_priority='1.0', position='75')>
```
Can we have
```
<QueueInfo(job_id='5e22013e69f2d4001830bbbd', _status='job is queued', estimated_start_time='2020-01-17T22:58:54.507000+00:00', estimated_complete_time='2020-01-17T22:59:25.907000+00:00', position='75', hub_priority='1.0', group_priority='1.0', project_priority='1.0')>
```

Reply:
Yeah, definitely! That'd be a nice feature. I'll go ahead and update the PR.

Reply:
Yeah definitely! That would be a nice feature to have. I'll go ahead and update the PR!